scaffolds/streamlit_on_cloudrun/llm_chatbot/build_app.ipynb

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
<p>This cell throws a NameError because "os" has still not been imported. import os here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks! Fixed :)</p>